lib/validate.js

```diff
@@ -96,8 +96,11 @@ module.exports = {
       _.merge(entries, entry);
     } else {
       _.forEach(functions, (func, index) => {
-        const entry = getEntryForFunction.call(this, functions[index], this.serverless.service.getFunction(func));
-        _.merge(entries, entry);
+        const functionDef = this.serverless.service.getFunction(func);
+        if (_.startsWith(functionDef.runtime, 'node')) {
```

Comment:
What happens when the global runtime is node and the runtime is not specified per function?